services/httpd/handler.go

```diff
@@ -257,23 +257,17 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, user *meta.
 	p := influxql.NewParser(strings.NewReader(qp))
 	db := q.Get("db")
 
+	// Sanitize the request query params so it doesn't show up in the response logger.
+	// Do this before anything else so a parsing error doesn't leak passwords.
+	sanitize(r)
```

Comment:
The response logger feels like a more natural fit for this code than the query handler. Other requests won't have a `q` query parameter but it could also be skipped with an `if r.URL.EscapedPath() == "/query"` check.